serialization/internal/TypeUtils.hx

```diff
@@ -212,4 +212,22 @@ class TypeUtils
 
     return null;
   }
+
+  public static function getEnumParameterCount(enm:Enum<Dynamic>, constructor:String) : Int {
+    #if neko
```

Comment:
I ended up using an actual enum value instance to get the parameter count. This is using code lifted from the old hxSerialize for enums. I'm concerned that the counts might change depending on optional enum value parameters.